### ISSUE TITLE
a11y: Color Contrast - Extensions

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13732]
Insufficient color contrast for informative image

## Actual results

"The informative image present on the page has an insufficient color contrast ratio with their background
making it difficult for low vision users to discern meaningful graphics that have insufficient contrast."

## Instances

"Extensions
Global Settings
Foreground Text Color: #3D98D3
Background Text Color: #E2F0F8
Ratio: 2.72:1 
Main Content
- More Actions Buttons (on default state)

"

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
-BUILT-IN
- Installed
- Available
- Updates 
- All

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- 1.0.0 (available after activating 
AKS Provisioning, EKS Provisioning, GKE Provisioning and 
 Virtualization Manager)  "

## Expected results

The contrast ratio between the key part of the image and its adjacent colors should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT

> "The below mentioned instance are not available:
> 
> Foreground Text Color: #3D98D3
> Background Text Color: #FFFFFF 
> Ratio: 3.16:1 
> Main Content
> - BUILT-IN
> - Installed
> - Available
> - Updates 
> - All"

Issue 216
Issue 217 - FIXED
